akinet/http/parser.go

```diff
@@ -14,6 +14,13 @@ import (
 	"github.com/akitasoftware/akita-libs/memview"
 )
 
+var (
+	// Default maximum HTTP length supported.
+	// Can be altered by the CLI as a configuration setting, but doing so after parsing
+	// has started will be a race condition.
+	MaximumHTTPLength int64 = 1024 * 1024
```

Comment:
Is the thread/goroutine setting this is the same as the one reading? If not, I think the Go memory model asks for a synchronization point before the effects of the write are guaranteed visible.

We could add a parameter to `NewHTTP{Request,Response}ParserFactory` and thread the value through from there. Takes effort, though.

Reply:
I guess we're fine, since this will be set during processing of command-line options, before any goroutines are created.

Reply:
Yes, as long as it's done before the apidump.Run() command starts we should be fine.